LICENSE

```diff
@@ -1,7 +1,7 @@
 The MIT License (MIT)
 
-Copyright (c) 2016-2018 Protocol Labs Inc.
-Copyright (c) 2018 Haja Networks Oy
+Copyright (c) 2016-2019 Protocol Labs Inc.
```

Comment:
Let's not update Protocol Labs copyright, just change Haja's to 2018-2019. Thanks.